alacritty/src/event.rs

```diff
@@ -726,7 +726,14 @@ impl<N: Notify + OnResize> Processor<N> {
         cli_options: CLIOptions,
     ) -> Processor<N> {
         #[cfg(not(any(target_os = "macos", windows)))]
-        let clipboard = Clipboard::new(display.window.wayland_display());
+        let clipboard = {
+            #[cfg(all(feature = "x11", not(feature = "wayland")))]
+            let wayland_display = None;
+            #[cfg(feature = "wayland")]
+            let wayland_display = display.window.wayland_display();
```

Comment:
What happens when we call this without Wayland feature enabled? Is it just not available at all?

It would probably be a bit easier and shorter to just have three statements with three annotations using `Clipboard::new(None)` and `Clipboard::new(display.window.wayland_display())` instead?

Reply:
I can return `None` form window.rs I guess, should be the same, but in a bit different place. If `Wayland` feature is disable you can't get `wayland_display` since no Wayland.

Reply:
Though, I'd refactor to use 3 statements that just create a clipboard.

Reply:
I think it might be nice to just handle these kind of things in window.rs when I think about it tbh. That way our callee never has to worry about a thing.